### ISSUE TITLE
Layer: Map of planning data for England

### ISSUE THREAD

**Issue by MKIM1008** [colouring-cities/colouring-london#186]
Adding layers referring to [Map of planning data for England ](https://www.planning.data.gov.uk/map/)

**Comment by polly64** [comment]
@taimaz22 and @MKIM1008 do you want to add all of these?

**Comment by MKIM1008** [comment]
We will review /share it with the team during the monthly meeting next week.

**Comment by taimaz22** [comment]
I agree with Polly that we can add all of them. We will double-check this with the rest of the group in the next meeting as well.
